ui/src/app/machines/components/MachineHeaderForms/ActionFormWrapper/TagForm/TagFormFields/TagFormFields.tsx

```diff
@@ -7,11 +6,21 @@ import tagSelectors from "app/store/tag/selectors";
 export const TagFormFields = (): JSX.Element => {
   const tags = useSelector(tagSelectors.all);
   return (
-    <Row>
-      <Col size={6}>
+    <div className="tag-form">
+      <div className="tag-form__search">
         <TagNameField required tagList={tags.map(({ name }) => name)} />
-      </Col>
-    </Row>
+      </div>
+      <div className="tag-form__changes">
```

Comment:
Can you think of a semantic element to be used here instead of a `div`? What about `<fieldset >`?